pkg/slo-controller/noderesource/plugins/batchresource/plugin.go

```diff
@@ -138,6 +138,10 @@ func (p *Plugin) Calculate(strategy *extension.ColocationStrategy, node *corev1.
 	nodeUsage := getNodeMetricUsage(nodeMetric.Status.NodeMetric)
 	systemUsed := quotav1.Max(quotav1.Subtract(nodeUsage, podAllUsed), util.NewZeroResourceList())
 
+	// System.Used = max(System.Used, Node.Anno.Reserved)
+	nodeAnnoReserved := util.GetNodeReservationFromAnnotation(node.Annotations)
+	systemUsed = quotav1.Max(systemUsed, nodeAnnoReserved)
```

Comment:
Could you please add a case in UT `TestPluginCalculate` where the node has annotation reserved resources to verify the calculation?

Reply:
done